src/rfac/rfsplint.c

```
@@ -0,0 +1,78 @@
+ * For a description see standard references on cubic spline interpolation.
+ *
+ *
+ * \return y interpolated intensity.
+ *
+ * \warning exits with \ref RFAC_BAD_IV_DATA if list is invalid.
+ *
+ */
+real rfac_splint(real eng, rfac_iv_data *list, size_t leng)
+{
+  real y;
+  int klo, khi, k;
```

Clang (14.0.6)

[warning] multiple declarations in a single statement reduces readability

(readability-isolate-declaration)